### ISSUE TITLE
Accessibility: Cannot scroll if SliverPersistentHeader is full height

### ISSUE THREAD

**Issue by lukepighetti** [flutter/flutter#97584]
## Steps to Reproduce

1. Turn on VoiceOver 
2. Execute `flutter run` on the code sample 
3. Select title or body, then three finger swipe to scroll 

**Expected results:** 

Scrollable

**Actual results:** 

Non-scrollable

https://user-images.githubusercontent.com/666539/152012670-832fda05-1422-4d77-a7f6-8073a9215c4d.MP4

<details>
<summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: const MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({Key? key, required this.title}) : super(key: key);

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Scrollbar(
        child: CustomScrollView(
          slivers: [
            SliverAppBar(
              title: Text("Title"),
              excludeHeaderSemantics: true,
              flexibleSpace: Container(
                color: Colors.red,
                alignment: Alignment.center,
                padding: EdgeInsets.all(25),
                child: Text(
                  'Select me with VoiceOver, '
                  'then three finger swipe to try to scroll down',
                ),
              ),
              expandedHeight: 1200,
            ),
            SliverList(
              delegate: SliverChildListDelegate.fixed(
                [
                  for (final i in List.generate(100, (index) => index))
                    ListTile(
                      title: Text('$i'),
                    ),
                ],
              ),
            ),
          ],
        ),
      ),
    );
  }
}
```

</details>

<details>
  <summary>Logs</summary>

```
```

```
```

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.8.1, on macOS 12.1 21C52 darwin-x64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.63.2)
[✓] Connected device (3 available)

• No issues found!
```

</details>

**Comment by darshankawar** [comment]
Thanks for the report. I do see the same behavior using latest stable and master. 

<details>
<summary> master, stable flutter doctor -v</summary>

```

dhs@Dhss-MacBook-Pro example % flutter doctor -v
[✓] Flutter (Channel master, 2.10.0-1.0.pre.369, on Mac OS X 10.15.4 19E2269 darwin-x64, locale en-GB)
    • Flutter version 2.10.0-1.0.pre.369 at /Users/dhs/documents/fluttersdk/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 9bdf809e53 (4 hours ago), 2022-02-01 22:25:19 -0500
    • Engine revision f22420acf7
    • Dart version 2.17.0 (build 2.17.0-69.0.dev)
    • DevTools version 2.10.0-dev.1

[!] Xcode - develop for iOS and macOS (Xcode 12.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    ! Flutter recommends a minimum Xcode version of 13.
      Download the latest version or update via the Mac App Store.
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] VS Code (version 1.62.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.21.0

[✓] Connected device (4 available)
    • Darshan's iphone (mobile)  • 21150b119064aecc249dfcfe05e259197461ce23 • ios            • iOS 14.4.1
      18D61
    • iPhone 12 Pro Max (mobile) • A5473606-0213-4FD8-BA16-553433949729     • ios            •
      com.apple.CoreSimulator.SimRuntime.iOS-14-3 (simulator)
    • macOS (desktop)            • macos                                    • darwin-x64     • Mac OS X
      10.15.4 19E2269 darwin-x64
    • Chrome (web)               • chrome                                   • web-javascript • Google Chrome
      97.0.4692.99

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.

[✓] Flutter (Channel stable, 2.8.1, on Mac OS X 10.15.4 19E2269 darwin-x64,
    locale en-GB)
    • Flutter version 2.8.1 at /Users/dhs/documents/fluttersdk/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 77d935af4d (6 weeks ago), 2021-12-16 08:37:33 -0800
    • Engine revision 890a5fca2e
    • Dart version 2.15.1

[!] Xcode - develop for iOS and macOS (Xcode 12.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    ! Flutter recommends a minimum Xcode version of 13.0.0.
      Download the latest version or update via the Mac App Store.
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] VS Code (version 1.62.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.21.0

[✓] Connected device (4 available)
    • Darshan's iphone (mobile)  • 21150b119064aecc249dfcfe05e259197461ce23 • ios            • iOS 14.4.1
      18D61
    • iPhone 12 Pro Max (mobile) • A5473606-0213-4FD8-BA16-553433949729     • ios            •
      com.apple.CoreSimulator.SimRuntime.iOS-14-3 (simulator)
    • macOS (desktop)            • macos                                    • darwin-x64     • Mac OS X
      10.15.4 19E2269 darwin-x64
    • Chrome (web)               • chrome                                   • web-javascript • Google Chrome
      97.0.4692.99

! Doctor found issues in 1 category.

**Comment by chunhtai** [comment]
looks like the parent semantics node of the label has scroll up action but the accessibility bridge for did not propagate the scroll up action to the parent. Need more investigation

**Comment by lukepighetti** [comment]
Is there any way to shim this in the meantime?
